corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -19,13 +19,14 @@
         attr: collapsible ? {
                 role: 'button',
                 'aria-expanded': showChildren() ? 'true' : 'false',
-                'aria-labelledby': captionId()
+                'aria-labelledby': captionId(),
+                tabindex: ko.utils.unwrapObservable($data.caption_markdown) || caption() ? '0' : '-1'
```

Comment:
by making `tabindex: -1` that means it's still available in the accessibility tree, correct? Versus using `aria-hidden="true"` removes it completely from the tree? I think for this use case it works, but wanted to flag in case one is now recommended over the other or if you had a strong feeling towards tabindex that I should know about

Reply:
yeah, with `tabindex: -1` means it will still be available in the accessibility tree, `tabindex: -1` is good here because if someone uses the <tab> key to navigate they wont be stuck on groups with no descriptions which are not read by screen-readers. `aria-hidden: true` will completely remove this element from the accessibility tree.

Reply:
Cool thanks for claifying